tests/foreman/ui_airgun/test_jobinvocation.py

```diff
@@ -163,12 +165,14 @@ def test_positive_run_custom_job_template_by_ip(
                 'search_query': 'name ^ {}'.format(hostname),
                 'template_content.command': 'ls',
             })
+            job_description = '{0} with inputs command="ls"'.format(
```

Comment:
no escaping with backslashes?

Reply:
The tests was run many many times and there was no problem found in that

Reply:
On the other hand - if tests are passing without it then we probably don't need it :)